rules/msk_module_backend.go

```diff
@@ -0,0 +1,148 @@
+package rules
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/gohcl"
+	"github.com/terraform-linters/tflint-plugin-sdk/hclext"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+)
+
+// MskModuleBackendRule checks whether an MSK module has an S3 backend defined with a key that has as suffix the name of the team.
+type MskModuleBackendRule struct {
+	tflint.DefaultRule
+}
+
+// NewMskModuleBackendRule returns a new rule.
+func NewMskModuleBackendRule() *MskModuleBackendRule {
+	return &MskModuleBackendRule{}
+}
+
+// Name returns the rule name.
+func (r *MskModuleBackendRule) Name() string {
+	return "msk_module_backend"
+}
+
+// Enabled returns whether the rule is enabled by default.
+func (r *MskModuleBackendRule) Enabled() bool {
+	return true
+}
+
+// Severity returns the rule severity.
+func (r *MskModuleBackendRule) Severity() tflint.Severity {
+	return tflint.ERROR
+}
+
+// Link returns the rule reference link.
+func (r *MskModuleBackendRule) Link() string {
+	return ""
```

Comment:
Would it be useful to link to this repo/the added README in this directory? (I'm assuming `Link` is an informative link that shows when the user sees this linter fail)

Reply:
Nice catch. I implemented it here: https://github.com/utilitywarehouse/tflint-ruleset-kafka-config/pull/5/commits/788cc41ffb813e379d07fae85deea595f69b6ba6

Based on the reference implementation from here: https://github.com/terraform-linters/tflint-ruleset-terraform/blob/main/rules/terraform_comment_syntax.go#L39 , but I dropped the version as that would need constant update, and just used the main blob reference